src/main/java/hudson/tasks/junit/pipeline/JUnitResultsStepExecution.java

```diff
@@ -70,7 +69,7 @@ protected TestResultSummary run() throws Exception {
         List<FlowNode> enclosingBlocks = new ArrayList<>();
         for (FlowNode enclosing : node.getEnclosingBlocks()) {
             if (enclosing != null && enclosing.getAction(LabelAction.class) != null) {
-                if ((enclosing instanceof StepStartNode && ((StepStartNode) enclosing).getDescriptor() instanceof StageStep.DescriptorImpl) ||
+                if ((enclosing instanceof StepNode && ((StepNode) enclosing).getDescriptor().getFunctionName().equals("stage")) ||
```

Comment:
Removing two unnecessary deps here. Cf. suggestions in #76 by both @svanoort and @jtnord, and my requests in https://github.com/jenkinsci/parallel-test-executor-plugin/pull/31#discussion_r145182474 and https://github.com/jenkinsci/parallel-test-executor-plugin/pull/31#discussion_r149539509.

(Also https://github.com/jenkinsci/parallel-test-executor-plugin/pull/31#discussion_r145182600: I am not sure why we even care whether this is a `stage` or something else with `LabelAction`? And why the separate check for `ThreadNameAction` when `parallel` block nodes will _also_ have `LabelAction`? Cf. https://github.com/jenkinsci/parallel-test-executor-plugin/pull/31#discussion_r149539657. Anyway, here I am not changing behavior, just removing unnecessary code.)